README.md

```diff
@@ -2,7 +2,7 @@
 
 ![n8n.io - Workflow Automation](https://raw.githubusercontent.com/n8n-io/n8n/master/docs/images/n8n-logo.png)
 
-n8n is a free node based "Open Source" (with Commons Clause)
+n8n is a free node based source-availalbe
```

Comment:
@ddevault typo: should be "source-available"

Reply:
Thanks, fixed.